src/render/program.js

```diff
@@ -198,7 +198,8 @@ class Program<Us: UniformBindings> {
          zoom: ?number,
          configuration: ?ProgramConfiguration,
          dynamicLayoutBuffer: ?VertexBuffer,
-         dynamicLayoutBuffer2: ?VertexBuffer) {
+         dynamicLayoutBuffer2: ?VertexBuffer,
+         dynamicLayoutBuffer3: ?VertexBuffer) {
```

Comment:
Not a blocker for merging and can be addressed separately, but we could start using an array here to simplify the redundant code binding these buffers.